bin/main.rb

```diff
@@ -14,14 +14,13 @@
 puts "| #{board[6]} | #{board[7]} | #{board[8]} |"
 puts '-------------'
 
-# Ask to Player 1 put X
-
+# until board.winner or board.draw
+# take input from player1
 puts 'Player 1, place X'
 player1 = gets.chomp.to_i
```

Comment:
- [optional] You should validate the user input (and display it the mistake)

Reply:
Oh yes, we understand. Thanks @expjazz